satpy/tests/test_config.py

```diff
@@ -111,7 +110,7 @@ def test_areas_rasterio(self):
                     # in a generic test so just skip this area
                     continue
             proj_dict = area_obj.proj_dict
-            if proj_dict['proj'] in ('ob_tran', 'nsper') and \
+            if proj_dict.get('proj') in ('ob_tran', 'nsper') and \
```

Comment:
Possibly missing default in `get` ?

Reply:
The default is 'None' if not specified, I though it was enough ?

Reply:
Of course, sorry!